library/Fission/Internal/Fixture/Web.hs

```diff
@@ -0,0 +1,36 @@
+module Fission.Internal.Fixture.Web 
+  ( fissionURL
+  , failureResp
+  , failure502
+  , failure504
+  ) where
+
+import Fission.Prelude
+
+import Servant.Client
+import Servant.Client.Core.Request
+import Servant.Client.Internal.HttpClient
+
+import Network.HTTP.Types.Status
+import Network.HTTP.Types.Version
+import Data.Sequence as Seq
+
+fissionURL :: BaseUrl
+fissionURL = BaseUrl (Https) "example.com" 443 "/"
+
+failureResp :: Status -> ClientError
+failureResp status = mkFailureResponse fissionURL defaultRequest $
+  Response 
+    { responseStatusCode = status
+    , responseHeaders = Seq.empty
+    , responseHttpVersion = http11
+    , responseBody = ""
+    }
```

Comment:
Instead of doing this by hand, you could use a helper function like https://www.stackage.org/haddock/lts-16.2/wai-3.2.2.1/Network-Wai.html#v:responseLBS

This would also remove the need for `containers`

Reply:
yeah the servant `Reponse` type (https://hackage.haskell.org/package/servant-client-core-0.17/docs/Servant-Client-Core.html#t:Response) is different from the Network.Wai `Response` type :unamused: 
I've been looking for a better way to do this

Reply:
Ah, okay I see. Yeah, this is probably intended by them to be internal.

`ServerError`s (e.g. `err404`) can be converted very easily with `responseServerError :: ServerError -> Response`.

Success cases lemme look 👀

Reply:
Okay, yeah you do need to build that manually.

On the plus side, we can get rid of the dependency by leaning into `RIO`:

https://www.stackage.org/haddock/lts-15.5/rio-0.1.14.1/RIO-Seq.html

Reply:
In general I'd say take a look at RIO first. It has a bunch of really useful stuff in there.

Reply:
ah good looks! that's perfect